packages/loopring_v3/contracts/impl/libexchange/ExchangeAccounts.sol

```diff
@@ -139,18 +142,19 @@ library ExchangeAccounts
 
     function updateAccount(
         ExchangeData.State storage S,
-        uint pubKeyX,
-        uint pubKeyY
+        address owner,
+        uint    pubKeyX,
+        uint    pubKeyY
         )
```

Comment:
The checks are in `ExchangeV3.sol`. The functions in the libraries are public/external but cannot be called directly by users. Well, I mean technically they can be, but they won't have an effect on the exchange state because they will operate on different storage.